native-engine/datafusion-ext-plans/src/agg/acc.rs

```diff
@@ -587,7 +587,7 @@ impl AccColumn for AccGenericColumn {
                     let len = read_len(&mut r)?;
                     if len > 0 {
                         let len = len - 1;
-                        let bytes = AccBytes::from({
+                        let bytes = AccBytes::from_vec({
```

Comment:
The `AccBytes::from` implementation also looks like a call to `AccBytes::from_vec`

Reply:
yes, we prefer to use a more explicit name.